libs/tools/release/src/changelog/changelog.ts

```diff
@@ -32,31 +32,36 @@ export async function prependChangelogFromLatestTag(
   });
 
   const newChangelogStream: any = getNewChangelog(headerPartial, releaseName);
-
   // Stream for reading the existing changelog. This is necessary because we want to
   // actually prepend the new changelog to the existing one.
-  const previousChangelogStream = createReadStream(changelogPath);
+  const previousChangelogStream: any = createReadStream(changelogPath);
```

Comment:
Please avoid the `any` because propper types help us finding bugs when upgrading dependencies and if they break something with their updates.

```suggestion
  const previousChangelogStream: ReadStream = createReadStream(changelogPath);
```

Reply:
Hi, yeah, this is something I don't like either. But further down I use ```readableEnded```, and ts thinks ```ReadStream``` does not have this property. This is why I used ```any```.

Reply:
but the `createReadStream` is from fs and the official node types for fs state that it is from the type `ReadStream` – so maybe investigate a little bit further.. have you tried it without nodejs streams?